src/java.base/share/classes/sun/nio/ch/FileChannelImpl.java

```diff
@@ -536,9 +536,9 @@ public void force(boolean metaData) throws IOException {
     }
 
     // Assume at first that the underlying kernel supports sendfile();
-    // set this to false if we find out later that it doesn't
+    // set this to true if we find out later that it doesn't
     //
-    private static volatile boolean transferSupported = true;
+    private static volatile boolean transferToNotSupported = false;
```

Comment:
The motive for inverting these flags was to avoid the volatile write, so you can drop "= false" from both flags now.